src/components/viewers/MultiFlatmap.vue

```diff
@@ -76,7 +82,13 @@ export default {
         payload: payload,
         type: this.entry.type,
       };
-      this.handleSyncPanZoomEvent(result)
+      let flatmapImp = this.getFlatmapImp();
+      let currentZoom = flatmapImp.getZoom()['zoom'];
+      if (this.zoomLevel !== currentZoom) {
+        this.zoomLevel = currentZoom;
+        this.flatmapMarkerZoomUpdate();
+      }
+      this.handleSyncPanZoomEvent(result);
```

Comment:
Do not call handleSyncPanZoomEvent here please, that method should be called only when it is triggered by external callback.